[docs] Add pricing links (#266)
mui-org/material-ui-x: docs/src/pages/components/data-grid/getting-started/getting-started.md

### Invalid license

> If you have an enterprise grid running with an expired or missing license key the grid displays a watermark, and a warning in shown in the console.

Reviewer: ```suggestion
If you have an enterprise grid running with an expired or missing license key the grid displays a watermark, and a warning is shown in the console.
```

(Sorry)

> If you have an enterprise grid running with an invalid license (no license, expired license, incorrect license) the grid displays a watermark and a warning in the console.

Reviewer: ```suggestion
If you have an enterprise grid running with an expired or missing license key the grid displays a watermark, and a warning in shown in the console.
```
